desktop/src/main/java/bisq/desktop/main/content/user/user_profile/UserProfileView.java

```diff
@@ -172,6 +173,18 @@ protected void onViewAttached() {
 
         selectedChatUserIdentityPin = EasyBind.subscribe(model.getSelectedUserIdentity(),
                 userIdentity -> comboBox.getSelectionModel().select(userIdentity));
+
+        isValidSelectionPin = EasyBind.subscribe(comboBox.getIsValidSelection(), isValidSelection -> UIThread.run(() -> {
+            if (!isValidSelection) {
+                statement.setEditable(false);
+                terms.setEditable(false);
+                controller.resetSelection();
+            }
+            else {
```

Comment:
the `else` should be in the same line as the closing }

Reply:
ok will update

Reply:
actually I had already changed it in https://github.com/bisq-network/bisq2/pull/1335/commits/8d2d61b0709d87cef5049f3e033cd2f945a301bd